src/iuse.h

```diff
@@ -35,7 +35,6 @@ class iuse
     int ecig                ( player*, item*, bool, const tripoint& );
     int antibiotic          ( player*, item*, bool, const tripoint& );
     int eyedrops            ( player*, item*, bool, const tripoint& );
-    int fungicide           ( player*, item*, bool, const tripoint& );
```

Comment:
Having a cheap ability to cure self of fungal infection is really useful, considering how anti-fun can they be.